### ISSUE TITLE
Why use dropout when inference?

### ISSUE THREAD

**Issue by lawo123** [TensorSpeech/TensorFlowTTS#749]
`self.f0_dropout(f0_embedding, training=True)`
Why use dropout when inference?
Will the model get better or worse if it is removed?
